C/cryptography/railFenceCipher.c

```diff
@@ -0,0 +1,64 @@
+/* The rail fence cipher (also called a zigzag cipher) is a form of transposition cipher. It derives its name from the way in which it is encoded. Given a plain-text message and a numeric key, it cipher the given text using Rail Fence algorithm(here key == 2). 
+*/
+#include<stdio.h>
+#include<string.h>
+  
+int main( void ) {
```

Comment:
Don't keep the primary logic of your code in the main() function. Use main() only for the driver code